ignite/handlers/early_stopping.py

```diff
@@ -31,19 +34,23 @@ def score_function(engine):
         evaluator.add_event_handler(Events.COMPLETED, handler)
 
     """
-    def __init__(self, patience, score_function, trainer):
+    def __init__(self, patience, score_function, trainer, min_delta=0.):
 
         if not callable(score_function):
             raise TypeError("Argument score_function should be a function.")
 
         if patience < 1:
             raise ValueError("Argument patience should be positive integer.")
 
+        if min_delta < 0.:
+            raise ValueError("Argument min_delta should not be a negative number.")
```

Comment:
Coverage patch complains about this line being not covered

Reply:
Test added. Thanks.